auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/GrpcUmsClient.java

```diff
@@ -39,15 +38,18 @@ private static String newRequestId() {
                         .build())) {
             UmsClient client = new UmsClient(channelWrapper.getChannel(), actorCrn);
             String requestId = newRequestId();
-            LOGGER.info(format("Getting user information for %s using request ID %s", userCrn, requestId));
+            LOGGER.info("Getting user information for {} using request ID {}", userCrn, requestId);
             UserManagementProto.User user = client.getUser(requestId, userCrn);
-            LOGGER.info("User information:");
-            LOGGER.info(user.toString());
+            LOGGER.info("User information retrieved for userCrn: {}", user.getCrn());
             return user;
         }
     }
 
     public boolean isConfigured() {
         return umsConfig.isConfigured();
     }
+
+    public boolean isUmsUsable(String crn) {
+        return umsConfig.isConfigured() && Crn.isValid(crn);
```

Comment:
This coupling seems a bit weird - or maybe it's the method name. I think that whether the ums is usable has nothing to do with the validity of the crn.

Reply:
Yes, but it represent the current situation. 
Cloudbreak has a UAA component that is used for service to service authentication in its own domain. This was the only Authentication system in place, then came caas but there was no service to service in caas, so UAA remained for that.

Now came the UMS, and we are halfway there.

This autentication implementation can also receive tokens that are not CRN's but UAA's token, so we only want to use UMS for Userinfo and authentication, if it's a valid CRN. There is another part in the code where we use the same condition, that's why I moved this it's own method.

Reply:
Also the CLI would stop working if we disabled the previous authentication method.